src/components/Header/Header.jsx

```diff
@@ -118,32 +116,41 @@ export const Header = (props) => {
                   </i>
                 </NavLink>
               </NavItem>
+              {(hasPermission(user.role, 'seeUserManagement') ||
+              hasPermission(user.role, 'seeBadgeManagement') ||
+              hasPermission(user.role, 'seeProjectManagement') ||
+              hasPermission(user.role, 'seeTeamsManagement') ||
+              hasPermission(user.role, 'seePopupManagement')) && (
               <UncontrolledDropdown nav inNavbar>
                 <DropdownToggle nav caret>
                   {OTHER_LINKS}
                 </DropdownToggle>
                 <DropdownMenu>
-                  {user.role === UserRole.Administrator ? (
+                  {hasPermission(user.role, 'seeUserManagement') ? (
                     <DropdownItem tag={Link} to="/usermanagement">
                       {USER_MANAGEMENT}
                     </DropdownItem>
                   ) : (
                     <React.Fragment></React.Fragment>
                   )}
-                  {user.role === UserRole.Administrator ? (
+                  {hasPermission(user.role, 'seeBadgeManagement') ? (
                     <DropdownItem tag={Link} to="/badgemanagement">
                       {BADGE_MANAGEMENT}
                     </DropdownItem>
                   ) : (
                     <React.Fragment></React.Fragment>
                   )}
-                  <DropdownItem tag={Link} to="/projects">
-                    {PROJECTS}
-                  </DropdownItem>
-                  <DropdownItem tag={Link} to="/teams">
-                    {TEAMS}
-                  </DropdownItem>
-                  {user.role === UserRole.Administrator ? (
+                  {hasPermission(user.role, 'seeProjectManagement') && (
```

Comment:
This catch is pretty nice! Even if now on beta site volunteer has `Projects` and `Teams` in `Other Links` dropdown part, based on the user class document they shouldn't be there.